wishlist-backend/wishlist/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+
+from wishlist.models import Wishlist
+from users.models import User
+
+from products.models import Product
+
+
+class WishlistSerializer(serializers.ModelSerializer):
+    products = serializers.PrimaryKeyRelatedField(many=True, queryset=Product.objects.all())
+    user = serializers.PrimaryKeyRelatedField(queryset=User.objects.all())
+
+    class Meta:
+        model = Wishlist
+        fields = ['id', 'products', 'user']
```

Comment:
Why is this a class in a class? we could have it in a separate model file?

Reply:
The class Meta thus as a configuration class for the serializer, providing metadata and instructions on how the serializer should map to the model and which fields it should include. This pattern is prevalent in Django, not just in serializers but also in forms, models, and admin configurations, offering a declarative way of specifying behavior and properties.

Reply:
I see.